src/main.c

```diff
@@ -2413,12 +2413,17 @@ void handle_movement(double dt) {
     State *s = &g->players->state;
     int sz = 0;
     int sx = 0;
+    int isRunning = 0;
     if (!g->typing) {
         float m = dt * 1.0;
         g->ortho = glfwGetKey(g->window, CRAFT_KEY_ORTHO) ? 64 : 0;
         g->fov = glfwGetKey(g->window, CRAFT_KEY_ZOOM) ? 15 : 65;
         if (glfwGetKey(g->window, CRAFT_KEY_FORWARD)) sz--;
         if (glfwGetKey(g->window, CRAFT_KEY_BACKWARD)) sz++;
+        if (glfwGetKey(g->window, CRAFT_KEY_FORWARD)) {
+            if(glfwGetKey(g->window, CRAFT_KEY_RUN)) isRunning = 1;
```

Comment:
I would suggest including this under the existing `CRAFT_KEY_FORWARD` if statement as it is duplicated code